core/src/main/kotlin/io/customer/sdk/communication/EventBus.kt

```diff
@@ -32,8 +39,7 @@ class EventBusImpl(
 
     val jobs = mutableListOf<Job>()
 
-    val scope: CoroutineScope = SDKComponent.androidSDKComponent?.scopeProvider?.eventBusScope
-        ?: CoroutineScope(Dispatchers.Default + SupervisorJob())
+    val scope: CoroutineScope = SDKComponent.androidSDKComponent?.scopeProvider?.eventBusScope ?: CoroutineScope(Dispatchers.Default + SupervisorJob())
```

Comment:
I think there should be consistent way to get scope here. As with this code, if the EventBus is somehow accessed before SDK is initialized, the scope would be different.

I just checked, and I think we can safely move the scope to `SDKComponent` for guaranteed access. Is there a reason for restricting it to `AndroidSDKComponent` that I missed? 🤔

Reply:
Makes sense, I wanted to keep it in `AndroidSDKComponent` so the modules who don't need it won't initialize then `AndroidSDKComponent` and we won't have this dependency added to them for no reason. But scope and dispatchers seems more of reusable component so happy to move there.